dace/codegen/CMakeLists.txt

```diff
@@ -428,58 +426,85 @@ if(DACE_ENABLE_XILINX)
 
   else()
 
-    add_custom_command(
-      OUTPUT ${DACE_PROGRAM_NAME}_sw_emu.xo
-      COMMAND XILINX_PATH=${CMAKE_BINARY_DIR} ${Vitis_COMPILER}
-      ${VITIS_BUILD_FLAGS}
-      -c
-      -t sw_emu
-      ${DACE_VITIS_KERNEL_FILES}
-      -o ${DACE_PROGRAM_NAME}_sw_emu.xo
-      DEPENDS ${DACE_VITIS_KERNEL_FILES})
+    # Since with Vitis only one kernel can be compiled at a time,
+    # for the general case we have first to compile each kernel
+    # and then link together the generated .xo files
```

Comment:
```suggestion
    # Since with Vitis only one kernel can be compiled at a time,
    # for the general case we have to first compile each kernel
    # and then link together the generated .xo files
```